File: BackEnd/Src/Conecta.API/Conecta.API/Models/Nota.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace Conecta.API.Models
{
    public class Nota
    {
        public int NotaId { get; set; }
        public int AlunoId { get; set; }
        public Aluno Aluno { get; set; }
        public decimal Bimestre1 { get; set; }
        public decimal Bimestre2 { get; set; }
        public decimal Bimestre3 { get; set; }
        public decimal Bimestre4 { get; set; }
        public DateTime Ano { get; set; }
        public int MateriaProfessorTurmaId { get; set; }
        public MateriaProfessorTurma MateriaProfessorTurma { get; set; }
    }
}

File: FrontEnd/Conecta.App/src/app/Components/shared/Nota.ts
import { materiaProfessorTurma } from './../materia-professor-turma/materiaProfessorTurma';
export class Nota {
  notaId: number;
  alunoId: number;
  bimestre1: number;
  bimestre2: number;
  bimestre3: number;
  bimestre4: number;
  ano: Date
  materiaProfessorTurmaId: number;
  materiaProfessorTurma: materiaProfessorTurma;
}
